### ISSUE TITLE
Output chimeras as junctions and BAM/SAM

### ISSUE THREAD

**Issue by kopardev** [alexdobin/STAR#1088]
Can you run `--chimOutType Junctions SeparateSAMold` with `--chimMultimapNmax 10`?

**Comment by alexdobin** [comment]
Hi Vishal,

```--chimOutType SeparateSAMold``` is the old chimeric output which is no longer supported, and so it will not work with newer chimeric options such as ```--chimMultimapNmax```.

Cheers
Alex

**Comment by kopardev** [comment]
Let me rephrase, I want to use the latest `--chimMultimapNmax 10` option and would like to subsequently visualize the "chimeric alignments only" in IGV. What is your recommendation for this?

**Comment by ChenhuaNJU** [comment]
I hava also the same question.Could you recommend for visualizing the 'chimeric alignments only' in IGV  in STAR?

**Comment by alexdobin** [comment]
Hi Vishal,  Chenhua,

with the 2.7.6a release, you can use ```--chimMultimapNmax 10 --chimOutType WithinBAM``` which will output chuimeras into the BAM file, in a "standard" format, which can be used for visualization. I do not think there is a special fusion display in IGV, but you may be able to hack something with the split-screen option. I also heard that JBrowse 2 has an option to display STAR-Fusion results (i.e. you do not even need the BAM) - but I have not used it.

Cheers
Alex

**Comment by DarioS** [comment]
I made a similar [feature request](https://github.com/igvteam/igv/issues/852) to IGV a couple of months ago for retrotransposon insertions. It can't come soon enough!
